guide/Overview

```diff
@@ -0,0 +1,57 @@
+
+### Converging new and traditional tech
+
+Today, the web has evolved to a point where traditional web technologies coexist alongside newer, secure and interactive technologies like blockchains or machine learning engines. 
+
+These emerging technologies have vast improvements over the traditional web but serve different purposes. Blockchains can be excellent for creating secure, trustless environments, but are notorious for UX challenges. Centralized systems, on the other hand, are often great for enabling the UX features the public now expect, but can leave user data vulnerable to attack or exploitation. 
+
+To be able to use technologies for their strengths when and where it makes sense, we need a way to run separate technologies in parallel to one another. The available tools that migrate traditional web services towards trustless, interactive services are currently costly, difficult to use and have many inherent limitations. Web service ecosystems help, but they are difficult to use and functionality is limited because they cannot connect to each other. 
+
+The goal of MESG is to connect traditional web service ecosystems to other systems like blockchains while maintaining the spirit of the newer trustless, decentralized solutions. Both old and new can exist harmoniously through applications built atop a decentralized infrastructure designed to support the bridging of traditional and emerging ecosystems.
+
+### A decentralized network of services 
+
+Some concepts in software and web development have proven to be more efficient than others. Service-Oriented Architectures (SOA) and Event-Driven Architecture (EDA) have become widely adopted because they are powerful and easy to scale. This inherent ease of scalability also makes these architectures easy to decentralize. 
+
+MESG combines SOA and EDA to form a decentralized network consisting of the following: 
+					
+						
+- **Engine** - The use and functionality of MESG’s decentralized network is primarily made possible via the Engine, a software that connects the actors in the network and manages the deployment of, and interaction between, services. 
+
+- **Services** - Services are discrete application components that make it possible to connect to specific business logic, a feature, or technology. Their bi-directional communication allows for the receiving of tasks to execute (like most classic Service-Oriented Frameworks) and the ability to proactively send events related to their business logic.
+
+- **Processes** - Processes are used to direct services, so applications can remain lightweight with sharable components. They consist of a list of tasks to be executed once a specific event occurs. A completed task can trigger one or more additional tasks depending on its result. 
+
+### Developer language SDKs
```

Comment:
I would remove the 2 paragraphs "Developer language SDKs" and "Running MESG on a local machine". They are too technical or are already explain in other pages.